manager-bundle/src/HttpKernel/ContaoKernel.php

```diff
@@ -319,17 +320,21 @@ private function getConfigFile(string $file): ?string
         $rootDir = $this->getProjectDir();
 
         foreach (['.yaml', '.yml'] as $ext) {
-            if (file_exists($rootDir.'/config/'.$file.$ext)) {
-                return $rootDir.'/config/'.$file.$ext;
+            $path = Path::join($rootDir, 'config', $file.$ext);
+
+            if (file_exists($path)) {
+                return $path;
```

Comment:
```suggestion
            if (file_exists($path = Path::join($rootDir, 'config', $file.$ext))) {
                return $path;
```
Just a minor nitpick 🤓

Reply:
I had this done already but got a (false positive) linter error and then did not find an easy way to work around. :roll_eyes: 

```
 Useless variable $path.

 Reported by:
"SlevomatCodingStandard\Sniffs\Variables\UselessVariableSniff.UselessVariable"
```

I changed it in a1fe9cb9afd683e9c8bc34e2cb2e1bcbeb24b284 for you to see :slightly_smiling_face:.

Reply:
We should disable or fix this linter then IMO. But I’m also OK with leaving it as is...

Reply:
See https://github.com/slevomat/coding-standard/pull/1053